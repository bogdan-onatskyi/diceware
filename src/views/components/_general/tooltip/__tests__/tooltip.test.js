import React from 'react';

import ToolTip from '../tooltip';

describe("ToolTip component", () => {
    it("should match saved snapshot", () => {
        const wrapper = shallow(
            <ToolTip/>
        );
        expect(wrapper).toMatchSnapshot();
    });
});
