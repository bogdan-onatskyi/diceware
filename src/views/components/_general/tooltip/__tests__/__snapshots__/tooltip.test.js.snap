// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ToolTip component should match saved snapshot 1`] = `
<ReactTooltip
  border={true}
  className="tooltip"
  delayShow={500}
  insecure={true}
  place="bottom"
  resizeHide={true}
  wrapper="div"
/>
`;
